src/memkind_memtier_mtt_allocator.c

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/* Copyright (C) 2022 Intel Corporation. */
+
+#include <memkind/internal/memkind_memtier_mtt_allocator.h>
+#include <memkind/internal/memkind_private.h>
+
+pthread_t bg_thread;
+
+static void *mtt_internal_create()
```

Comment:
Are these functions used? I think they are not required for now, they would be handled in separate file (mtt_internals), so we can delete them now